fury/actor.py

```diff
@@ -1719,6 +1719,46 @@ def triangularprism(centers, directions=(1, 0, 0), colors=(1, 0, 0),
     return tri_actor
 
 
+def pentagonalprism(centers, directions=(1, 0, 0), colors=(1, 0, 0), scales=1):
+    """Visualize one or many pentagonal prisms with different features.
+
+    Parameters
+    ----------
+    centers : ndarray, shape (N, 3), optional
+        Pentagonal prism positions
+    directions : ndarray, shape (N, 3), optional
+        The orientation vector of the pentagonal prism.
+    colors : ndarray (N,3) or (N, 4) or tuple (3,) or tuple (4,), optional
+        RGB or RGBA (for opacity) R, G, B and A should be at the range [0, 1]
+    scales : int or ndarray (N,3) or tuple (3,), optional
+        Pentagonal prism size on each direction (x, y), default(1)
+
+    Returns
+    -------
+    vtkActor
+
+    Examples
+    --------
+    >>> from fury import window, actor
+    >>> scene = window.Scene()
+    >>> centers = np.random.rand(3, 3)
+    >>> dirs = np.random.rand(3, 3)
+    >>> colors = np.random.rand(3, 3)
+    >>> scales = np.random.rand(3, 1)
+    >>> actor = actor.pentagonalprism(centers, dirs, colors, scales)
```

Comment:
You should avoid to overwrite an import.  In your case you have the same name for actor which is a module from FURY and the pentagonalprism. I think the following is better
``python
from fury import window, actor
....
 actor_pentagonal = actor.pentagonalprism(centers, dirs, colors, scales)
 ```